nmz/cli/inspectors/proc.go

```diff
@@ -63,9 +67,29 @@ func (cmd procCmd) Run(args []string) int {
 		return 1
 	}
 
+	var pid int
 	if _procFlags.RootPID <= 0 {
-		log.Critical("pid is not set (or set to non-positive value)")
-		return 1
+		if 0 < len(_procFlags.Cmd) {
+			var err error
+
+			args := strings.Split(_procFlags.Cmd, " ")
+			proc, err := os.StartProcess(args[0], args[1:], &os.ProcAttr{})
```

Comment:
Maybe `os/exec.Command` is preferred?

Example
http://qiita.com/tanksuzuki/items/9205ff70c57c4c03b5e5#%E5%AE%9F%E8%A1%8C%E3%81%97%E3%81%A6%E3%81%84%E3%82%8B%E3%81%97%E3%81%9F%E3%82%B3%E3%83%9E%E3%83%B3%E3%83%89%E3%81%AEpid%E3%82%92%E5%8F%96%E5%BE%97%E3%81%99%E3%82%8B